src/routes/+page.svelte

```diff
@@ -1,2 +1,123 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script>
+  import Masthead from '$lib/masthead.svelte';
+</script>
+
+<Masthead />
+
+<style global>
```

Comment:
I think we already have global styles in their own file, so when we merge this PR we should probably remove them from the page.svelte file to avoid duplicating them.

Reply:
I think I removed the duplicate styling. lmk if it's what you expected. 👍

Reply:
Yeah, looks good!